### PR TITLE
Bump astrochem manifest

### PR DESCRIPTION
**Pull request by ChrisRackauckas**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
```julia
Error: MethodError: no method matching jacobian2W!(::Matrix{Float64}, ::Lin
earAlgebra.UniformScaling{Bool}, ::Float64, ::Matrix{Float64})

Closest candidates are:
  jacobian2W!(::Matrix, ::Any, ::Number, ::Matrix, !Matched::Bool)
   @ OrdinaryDiffEqDifferentiation /cache/julia-buildkite-plugin/depots/5b3
00254-1738-4989-ae0a-f4d2d937f953/packages/OrdinaryDiffEqDifferentiation/wk
zwo/src/derivative_utils.jl:574
  jacobian2W!(::AbstractMatrix, ::Any, ::Number, ::AbstractMatrix, !Matched
::Bool)
   @ OrdinaryDiffEqDifferentiation /cache/julia-buildkite-plugin/depots/5b3
00254-1738-4989-ae0a-f4d2d937f953/packages/OrdinaryDiffEqDifferentiation/wk
zwo/src/derivative_utils.jl:491
```

@oscardssmith do you know the cause of this?

**Comment by oscardssmith** [comment]
it means you don't have updated versions. we probably need to add some more compats

**Comment by ChrisRackauckas** [comment]
No it's all latest in the manifest:

```
[1dea7af3] OrdinaryDiffEq v6.89.0
  [89bda076] OrdinaryDiffEqAdamsBashforthMoulton v1.1.0
  [6ad6398a] OrdinaryDiffEqBDF v1.1.2
  [bbf590c4] OrdinaryDiffEqCore v1.9.0
  [50262376] OrdinaryDiffEqDefault v1.1.0
  [4302a76b] OrdinaryDiffEqDifferentiation v1.1.0
  [9286f039] OrdinaryDiffEqExplicitRK v1.1.0
  [e0540318] OrdinaryDiffEqExponentialRK v1.1.0
  [becaefa8] OrdinaryDiffEqExtrapolation v1.2.0
  [5960d6e9] OrdinaryDiffEqFIRK v1.2.0
  [101fe9f7] OrdinaryDiffEqFeagin v1.1.0
  [d3585ca7] OrdinaryDiffEqFunctionMap v1.1.1
  [d28bc4f8] OrdinaryDiffEqHighOrderRK v1.1.0
  [9f002381] OrdinaryDiffEqIMEXMultistep v1.1.0
  [521117fe] OrdinaryDiffEqLinear v1.1.0
  [1344f307] OrdinaryDiffEqLowOrderRK v1.2.0
  [b0944070] OrdinaryDiffEqLowStorageRK v1.2.1
  [127b3ac7] OrdinaryDiffEqNonlinearSolve v1.2.1
  [c9986a66] OrdinaryDiffEqNordsieck v1.1.0
  [5dd0a6cf] OrdinaryDiffEqPDIRK v1.1.0
  [5b33eab2] OrdinaryDiffEqPRK v1.1.0
  [04162be5] OrdinaryDiffEqQPRK v1.1.0
  [af6ede74] OrdinaryDiffEqRKN v1.1.0
  [43230ef6] OrdinaryDiffEqRosenbrock v1.2.0
  [2d112036] OrdinaryDiffEqSDIRK v1.1.0
  [669c94d9] OrdinaryDiffEqSSPRK v1.2.0
  [e3e12d00] OrdinaryDiffEqStabilizedIRK v1.1.0
  [358294b1] OrdinaryDiffEqStabilizedRK v1.1.0
  [fa646aed] OrdinaryDiffEqSymplecticRK v1.1.0
  [b1df2697] OrdinaryDiffEqTsit5 v1.1.0
  [79d7bb75] OrdinaryDiffEqVerner v1.1.1
  ```

**Comment by oscardssmith** [comment]
no, but the failure is completely different. I'll investigate.

**Comment by ChrisRackauckas** [comment]
@oscardssmith how has this gone?

File: benchmarks/AstroChem/astrochem.jmd [hunk 1]
```diff
@@ -392,14 +392,13 @@ reltols = 1.0 ./ 10.0 .^ (9:10)
...
+          #Dict(:alg=>QNDF()),
           Dict(:alg=>CVODE_BDF()),
           #Dict(:alg=>ddebdf()),
           #Dict(:alg=>Rodas4()),
           Dict(:alg=>Rodas5P()),
-          Dict(:alg=>KenCarp4()),
-          Dict(:alg=>KenCarp47()),
+          #Dict(:alg=>KenCarp4()),
+          #Dict(:alg=>KenCarp47()),
           Dict(:alg=>RadauIIA9()),
```
Comment (ChrisRackauckas) [inline benchmarks/AstroChem/astrochem.jmd:402]:
```suggestion
          #Dict(:alg=>RadauIIA9()),
```

File: benchmarks/AstroChem/astrochem.jmd [hunk 2]
```diff
@@ -381,9 +381,7 @@ refsol = solve(oprob, Rodas5P(), abstol=1e-14, reltol=1e-14)
 ```
 
 ```julia
-refsol = solve(oprob, Rodas5P(), abstol=1e-13, reltol=1e-13)
-
-solve(oprob, RadauIIA9(), abstol=1e-9, reltol=1e-9)
+refsol = solve(oprob, Rodas5P(), abstol=1e-13, reltol=1e-13))
```
Comment (ChrisRackauckas) [inline benchmarks/AstroChem/astrochem.jmd:384]:
```suggestion
refsol = solve(oprob, Rodas5P(), abstol=1e-13, reltol=1e-13)
```

File: benchmarks/AstroChem/astrochem.jmd [hunk 3]
```diff
@@ -393,17 +391,15 @@ reltols = 1.0 ./ 10.0 .^ (9:10)
...
           Dict(:alg=>FBDF()),
           Dict(:alg=>QNDF()),
-          #Dict(:alg=>Rodas4P()),
           Dict(:alg=>CVODE_BDF()),
           #Dict(:alg=>ddebdf()),
-          #Dict(:alg=>Rodas4()),
           Dict(:alg=>Rodas5P()),
           Dict(:alg=>KenCarp4()),
           Dict(:alg=>KenCarp47()),
           Dict(:alg=>RadauIIA9()),
```
Comment (ChrisRackauckas) [inline benchmarks/AstroChem/astrochem.jmd:399]:
```suggestion
          #Dict(:alg=>RadauIIA9()),
```

**Pull request merged**